container/build.go

```diff
@@ -33,15 +33,17 @@ func (c *DockerContainer) Build(path string) (tag string, err error) {
 		return "", err
 	}
 	defer buildContext.Close()
+	tag = name + ":" + version
```

Comment:
what is the limitation on docker tag length and characters?

Reply:
also, i suggest to add the prefix `mesg/` or `mesg-service/` (and create the org on docker hub) so we prevent people to upload service image on docker hub (because they will not have the org access).

Reply:
from https://docs.docker.com/engine/reference/commandline/tag/

A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.

Reply:
so do we prepend `mesg` in that case ?
We have sid `63` and hash `64` plus the separator `:` we have a total of 128 but more if we add the `mesg`

Reply:
only 44 character for the hash if it's in base58 ;)

Reply:
so i'll keep the `mesg/` and we will change the representation of the hash in `base58` :)